shell/platform/tizen/channels/key_event_channel.cc

```diff
@@ -230,25 +283,37 @@ KeyEventChannel::~KeyEventChannel() {}
 void KeyEventChannel::SendKeyEvent(Ecore_Event_Key* key,
                                    bool is_down,
                                    std::function<void(bool)> callback) {
-  int gtk_keycode = 0;
-  if (kKeyCodeMap.count(key->keycode) > 0) {
-    gtk_keycode = kKeyCodeMap.at(key->keycode);
+  uint32_t scan_code = key->keycode;
+  auto iter = kSymbolToScanCode.find(key->key);
+  if (iter != kSymbolToScanCode.end()) {
+    scan_code = iter->second;
```

Comment:
This remapping is unavoidable because TV devices use their own key layout. For example, `XF86SysMenu` on TV remote control is defined as `0x00000085` (Meta Left).